src/lamplib/src/genny/cli.py

```diff
@@ -522,14 +522,22 @@ def lint_yaml(ctx: click.Context):
 @click.option(
     "--tasks", required=True, type=click.Choice(["all_tasks", "variant_tasks", "patch_tasks"]),
 )
+@click.option(
+    "-w",
+    "--workload-repo",
```

Comment:
Seems like genny should just look in all workload repos for genny tasks - I think that's what it's currently doing?

Reply:
As discussed. Every workload repo will have its own task generator including genny workloads